topics/ui/controls/progress_bar.md

```diff
@@ -21,112 +21,173 @@ Follow the rules for [progress indicators](progress_indicators.md).
-
-![](tasks_dialog.png){width=700}
-
-Always place the Stop icon on the right next to the progress bar. On hover over the Stop icon, show the "Stop" or "Cancel" comment under the progress bar instead of process details:
-
-![](hover_stop_icon.png){width=332}
-
-If the process takes a long time and can prevent the user from performing tasks, provide an option to pause a process using the Pause button or the Pause icon.
-Replace process details with the "Pause" comment on hover over the Pause icon:
-
-![](pause.png){width=331}
+#### Stop icon
+Use the Stop icon when multiple processes are running simultaneously in one dialog or when there isn’t enough space to display individual buttons (e.g., [Status bar](#in-status-bar)):
+
+![](16 process_control.png){width=706}
+
+#### Placement and Behavior
+<table style="none" border="false">
+  <tr>
+    <td width="378">
+      <img src="17 stop_button.png" alt="Stop icon on the progress bar"/>
+    </td>
+    <td>
+      <p>Always place the Stop icon on the right next to the progress bar. On hover over the Stop icon, show the "Stop" or "Cancel" comment under the progress bar instead of process details.</p>
+    </td>
+  </tr>
+  <tr>
+    <td width="378">
+      <img src="17 pause_button.png" alt="Pause button on the progress bar"/>
+    </td>
+    <td>
+      <p>If the process takes a long time and can prevent the user from performing tasks, provide an option to pause a process using the Pause button or the Pause icon.</p>
+    </td>
+  </tr>
+    <tr>
+     <td width="378">
+         <img src="18 pause_button.png" alt="Pause under the progress bar with hover effect"/>
+        </td>
+        <td>
+        <p>Replace process details with the "Pause" comment on hover over the Pause icon.</p>
+     </td>
+    </tr>
+    <tr>
+     <td width="378">
+         <img src="19 resume_button.png" alt="Resume button on the progress bar"/>
+        </td>
+        <td>
+        <p>If a user pauses the process, show "Paused" under the progress bar. Replace the Pause icon with Resume.</p>
+     </td>
+    </tr>
+    <tr>
+     <td width="378">
+         <img src="20 resume_button.png" alt="Resume under the progress bar with hover effect."/>
+        </td>
+        <td>
+        <p>Show "Resume" under the progress bar and when hovered over the Resume button.</p>
+     </td>
+    </tr>
+</table>
+
+#### Pause Option Recommendations
 
 It is **not** recommended providing an option to pause the process. It is preferable that the process runs in the background and does not interfere with a user.
 
-If a user pauses the process, show "Paused" under the progress bar.
-Replace the Pause icon with Resume, show "Resume" under the progress bar and when hovered over the Resume button:
-
-![](resume.png){width=330}
-
+#### Process Completion
 Hide the progress bar as soon as the process completes.
 
 ## Sizes and placement
```

Comment:
Let's remove this part as we did for other articles. It's easier to keep spacings in the UI kit, and sizes are built-in into controls themselves